adapter/internal/discovery/xds/marshaller.go

```diff
@@ -377,7 +377,7 @@ func MarshalKeyManager(keyManager *types.KeyManager) *keymgt.KeyManagerConfig {
 		}
 		return newKeyManager
 	}
-	logger.LoggerXds.Debugf("Error happens while marshaling key manager data for " + fmt.Sprint(keyManager.Name))
+	logger.LoggerXds.Debugf("Error happens while marshaling key manager data for %v", keyManager.Name)
```

Comment:
Shall we make this an error log. I think it was a mistake to have it as debug log.